internal/pubsub/gcppubsub/gcppubsub_test.go

```diff
@@ -72,9 +76,17 @@ func (h *harness) MakeNonexistentTopic(ctx context.Context) (driver.Topic, error
 	return dt, nil
 }
 
-func (h *harness) MakeSubscription(ctx context.Context, dt driver.Topic) (driver.Subscription, error) {
-	ds := openSubscription(ctx, h.subClient, projectID, subscriptionName)
-	return ds, nil
+func (h *harness) MakeSubscription(ctx context.Context, dt driver.Topic, n int) (driver.Subscription, error) {
+	var sname string
+	switch n {
+	case 0:
```

Comment:
Why not make these numbers match the names?

Reply:
Done.